### PR TITLE
Northwest Class 4 -Maha Malik - HTML/CSS - Week 2

### PR DESCRIPTION
**Pull request by maham511**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Maha Malik
- Your City: NorthWest
- Your Slack Name: Maha Malik

# Homework Details

- Module: CSS/HTML
- Week: 2

# Notes

- What did you find easy?
Structuring html

- What did you find hard?
CSS Layout, positions of elements

- What do you still not understand?
Fine-tune cropping an image (hero image), scaling and reposition it to fit container.

- Any other notes?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,83 @@
...
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
+    <link
```
Comment (lannem) [inline index.html:10]: nice formatting on this link tag! Makes the code much easier to read

File: index.html [hunk 2]
```diff
@@ -1,19 +1,83 @@
...
+        <div class="feature">
+          <img src="img/icon-coffee.svg">
+          <p>Boost your productivity</p>
+        </div>
+
+        <div class="feature">
+          <img src="img/icon-refill.svg">
+          <p>Pay as You Go</p>
+        </div>
+      </section>
```
Comment (lannem) [inline index.html:66]: You are missing a closing <section> tag somewhere from either line 49 or 51

File: css/style.css [hunk 3]
```diff
@@ -16,4 +15,201 @@ body {
...
+nav ul {
+  display: flex;
+  list-style-type: none;
+  padding: 0 80px 0 0;
+  position: absolute;
+}
+
+nav ul li {
+  padding: 20px;
+  /*  display: none; */
```
Comment (lannem) [inline css/style.css:55]: it is usually good practice to remove any commented out code

**Comment by maham511** [comment]
Thanks for the feedback, Lucy! I'll have a look at the missing closing tag.
Will also remove the unused code too, creature of habit 😅

**Pull request closed without merging**